parachains/runtimes/bridge-hubs/bridge-hub-kusama/src/lib.rs

```diff
@@ -589,14 +599,85 @@ impl_runtime_apis! {
 		fn dispatch_benchmark(
 			config: frame_benchmarking::BenchmarkConfig
 		) -> Result<Vec<frame_benchmarking::BenchmarkBatch>, sp_runtime::RuntimeString> {
-			use frame_benchmarking::{Benchmarking, BenchmarkBatch, TrackedStorageKey};
+			use frame_benchmarking::{Benchmarking, BenchmarkBatch, BenchmarkError, TrackedStorageKey};
 
 			use frame_system_benchmarking::Pallet as SystemBench;
 			impl frame_system_benchmarking::Config for Runtime {}
 
 			use cumulus_pallet_session_benchmarking::Pallet as SessionBench;
 			impl cumulus_pallet_session_benchmarking::Config for Runtime {}
 
+			use xcm::latest::prelude::*;
+			use xcm_config::KsmRelayLocation;
+
+			impl pallet_xcm_benchmarks::Config for Runtime {
+				type XcmConfig = xcm_config::XcmConfig;
+				type AccountIdConverter = xcm_config::LocationToAccountId;
+				fn valid_destination() -> Result<MultiLocation, BenchmarkError> {
+					Ok(KsmRelayLocation::get())
+				}
+				fn worst_case_holding() -> MultiAssets {
+					// just concrete assets according to relay chain.
+					let assets: Vec<MultiAsset> = vec![
+						MultiAsset {
+							id: Concrete(KsmRelayLocation::get()),
+							fun: Fungible(1_000_000 * UNITS),
+						}
+					];
+					assets.into()
+				}
+			}
+
+			parameter_types! {
+				pub const TrustedTeleporter: Option<(MultiLocation, MultiAsset)> = Some((
+					KsmRelayLocation::get(),
+					MultiAsset { fun: Fungible(1 * UNITS), id: Concrete(KsmRelayLocation::get()) },
+				));
+				pub const TrustedReserve: Option<(MultiLocation, MultiAsset)> = None;
```

Comment:
It's not recognizing the assets parachain as a trusted reserve?

Reply:
iiuc, we dont need to process any reserves from statemint/e on bridge-hub,

as I see `TrustedReserve` is used in benchmark `reserve_asset_deposited`,
and probably we dont expect to process `ReserveAssetDeposited` instruction on bridge-hub,
bridge-hub will have sovereign account on statemint/e, where reserves for moving asset throught bridge will be done, right?

@joepetrowski Joe, wdyt?

Reply:
Correct, BH shouldn't need to process `ReserveAssetDeposited`, it should just be a message forwarding/routing service. We actually don't really expect users (i.e. accounts with cryptographic keys) to transact on the BH, it would mostly be receiving messages from its sibling bridges or sibling paras.

Also, we still don't have any bridge/asset functionality in this runtime, so we don't want users to send assets here.